src/app/crawlstart/crawlstart.component.html

```diff
@@ -133,15 +133,13 @@ <h3 style="color: #dd4b39;font-family: Arial,SansSerif">Expert Crawl Start</h3>
         </div>
       </div>
       <div class="form-group">
-        <div class="col-md-4"><label for="safe search">Must-Match List for Country Codes
-          info </label>
+        <div class="col-md-4">Must-Match List for Country Codes
         </div>
         <div class="col-md-6">
-          <input name="country" [(ngModel)]="crawlvalues.countryMustMatchSwitch" type="radio" id="safe search">
+          <input name="countryMustMatchSwitch" type="radio" [(ngModel)]="crawlvalues.countryMustMatchSwitch" value="0" [checked]="crawlvalues.countryMustMatchSwitch==='0'">
           no country code restriction
-          <input name="country" [(ngModel)]="crawlvalues.countryMustMatchSwitch" type="radio">Use filter
-          <input name="country" [(ngModel)]="crawlvalues.countryMustMatchList" class="form-control" type="text" size="60" maxlength="256">
-        </div>
+          <input name="countryMustMatchSwitch" [(ngModel)]="crawlvalues.countryMustMatchSwitch" id="countryMustMatchSwitch" value="1" type="radio" >Use filter
+          <input name="countryMustMatchList" [(ngModel)]="crawlvalues.countryMustMatchList" id="countryMustMatchList" type="text" size="60" maxlength="256" value="AD,AL,AT,BA,BE,BG,BY,CH,CY,CZ,DE,DK,EE,ES,FI,FO,FR,GG,GI,GR,HR,HU,IE,IM,IS,IT,JE,LI,LT,LU,LV,MC,MD,MK,MT,NL,NO,PL,PT,RO,RU,SE,SI,SJ,SK,SM,TR,UA,UK,VA,YU">        </div>
```

Comment:
@Marauderer97 please remove this default value from HTML and add it to crawlvalues object in the component. this is because in future we don't pump default values from susper, we will be receiving them from Yacy crawler. you could find an already implemented function in the component to retrieve the default values from yacy which is commented out.